<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductEdit</class>
 <widget class="QWidget" name="ProductEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2010</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>商品编辑</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 全局样式：白色背景，深灰色字体 */
QWidget {
    background-color: #FFFFFF;
    font-size: 20px;
    color: #333333;
    font-family: &quot;Segoe UI&quot;, sans-serif;
}

/* QLabel 样式 */
QLabel {
    font-size: 20px;
    color: #333333;
    padding: 5px;
}

/* QLineEdit 样式 */
QLineEdit {
    background-color: #F5F5F5;
    border: 1px solid #1E90FF; /* 天蓝色边框 */
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus {
    border: 2px solid #63B8FF; /* 聚焦时更亮的天蓝色 */
}

/* QTextEdit 样式 */
QTextEdit {
    background-color: #F5F5F5;
    border: 1px solid #1E90FF; /* 天蓝色边框 */
    border-radius: 5px;
    padding: 5px;
}

QTextEdit:focus {
    border: 2px solid #63B8FF; /* 聚焦时更亮的天蓝色 */
}

/* QSpinBox 样式 */
QSpinBox {
    background-color: #FFFFFF;
    border: 1px solid #1E90FF; /* 天蓝色边框 */
    border-radius: 5px;
    padding: 2px 5px;
}

QSpinBox::up-button, QSpinBox::down-button {
    background-color: #1E90FF; /* 天蓝色按钮 */
    border: none;
    border-radius: 3px;
    margin: 1px;
}

QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: #63B8FF; /* 悬浮时更亮的天蓝色 */
}

QSpinBox::up-arrow, QSpinBox::down-arrow {
    width: 10px;
    height: 10px;
    image: url(:/Icons/up-arrow.png); /* 替换为你实际的图标路径 */
}

/* QPushButton 样式 */
QPushButton {
    background-color: #1E90FF; /* 天蓝色背景 */
    color: white;
    border-radius: 5px;
    padding: 10px 15px;
    font-size: 20px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #63B8FF; /* 悬浮时更亮的天蓝色 */
}

QPushButton:pressed {
    background-color: #4682B4; /* 按下时更深的蓝色 */
}

/* 返回按钮样式（小按钮） */
QPushButton#backButton {
    background-color: transparent;
    color: #1E90FF; /* 天蓝色字体 */
    font-size: 18px;
    border: none;
    text-align: left;
}

QPushButton#backButton:hover {
    color: #63B8FF; /* 悬浮时更亮的天蓝色 */
    text-decoration: underline;
}

/* QLabel 的图片容器样式 */
QLabel#productImageLabel {
    border: 1px solid #DDDDDD; /* 灰色边框 */
    background-color: #F5F5F5; /* 浅灰色背景 */
    border-radius: 5px;
    font-size: 20px;
    color: #999999; /* 灰色字体 */
    text-align: center;
}

/* QFormLayout 表单标签样式 */
QFormLayout &gt; QLabel {
    font-weight: bold;
}

/* QVBoxLayout 或 QHBoxLayout 的子控件间距 */
QVBoxLayout &gt; QWidget, QHBoxLayout &gt; QWidget {
    margin: 5px;
    padding: 5px;
}

/* QSpacerItem 间隔透明 */
QSpacerItem {
    background-color: transparent;
}

/* QSpinBox 样式 */
QSpinBox {
    background-color: #FFFFFF;
    border: 1px solid #1E90FF; /* 天蓝色边框 */
    border-radius: 5px;
    padding: 5px;
}

/* 图片选择按钮样式 */
QPushButton#selectImageButton {
    background-color: #1E90FF;
    color: white;
    border-radius: 5px;
    padding: 8px 15px;
    font-size: 20px;
    font-weight: bold;
}

QPushButton#selectImageButton:hover {
    background-color: #63B8FF;
}

/* 保存按钮样式 */
QPushButton#saveButton {
    background-color: #1E90FF;
    color: white;
    border-radius: 5px;
    padding: 8px 15px;
    font-size: 20px;
    font-weight: bold;
}

QPushButton#saveButton:hover {
    background-color: #63B8FF;
}

/* 滚动条样式 */
QScrollBar:vertical {
    background: #F5F5F5;
    width: 12px;
    margin: 0px;
}

QScrollBar::handle:vertical {
    background: #1E90FF;
    border-radius: 6px;
}

QScrollBar::handle:vertical:hover {
    background: #63B8FF;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: transparent;
    border: none;
    height: 0px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="saveButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>← 返回</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="headerSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="styleLayout">
     <item>
      <layout class="QHBoxLayout" name="styleHeaderLayout">
       <item>
        <widget class="QLabel" name="styleLabel">
         <property name="text">
          <string>款式数量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="styleCountSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="styleEditLayout"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="imageInfoLayout">
     <item>
      <layout class="QVBoxLayout" name="imageLayout">
       <item>
        <widget class="QLabel" name="productImageLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>商品图片</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectImageButton">
         <property name="text">
          <string>选择图片</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="infoLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>商品名称：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="nameEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="priceLabel">
         <property name="text">
          <string>价格：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="priceEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="stockLabel">
         <property name="text">
          <string>库存：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="stockSpinBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>描述：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="descriptionEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
